packages/flutter/test/cupertino/sliding_segmented_control_test.dart

```diff
@@ -2126,4 +2126,67 @@ void main() {
 
     expect(onValueChangedCalled, 0);
   });
+
+  testWidgets('CupertinoSlidingSegmentedControl can be momentary', (WidgetTester tester) async {
+    const Map<int, Widget> children = <int, Widget>{0: Text('A'), 1: Text('BB'), 2: Text('CCCC')};
+
+    groupValue = 1;
+    await tester.pumpWidget(
+      boilerplate(
+        builder: (BuildContext context) {
+          return CupertinoSlidingSegmentedControl<int>(
+            isMomentary: true,
+            children: children,
+            groupValue: groupValue,
+            onValueChanged: defaultCallback,
+          );
+        },
+      ),
+    );
+
+    expect(getHighlightedIndex(tester), null);
+
+    // Tap first segment.
+    await tester.tap(find.text('A'));
+    await tester.pumpAndSettle();
+
+    expect(getHighlightedIndex(tester), null); // The highlighted index doesn't change
+  });
+
+  testWidgets('CupertinoSlidingSegmentedControl with momentary scales up selected segment', (
+    WidgetTester tester,
+  ) async {
+    const Map<int, Widget> children = <int, Widget>{0: Text('A'), 1: Text('BB'), 2: Text('CCCC')};
+
+    groupValue = 1;
+    await tester.pumpWidget(
+      boilerplate(
+        builder: (BuildContext context) {
+          return CupertinoSlidingSegmentedControl<int>(
+            isMomentary: true,
+            children: children,
+            groupValue: groupValue,
+            onValueChanged: defaultCallback,
+          );
+        },
+      ),
+    );
+
+    final TestGesture gesture = await tester.startGesture(tester.getCenter(find.text('A')));
+    final Finder scaleTransition = find.ancestor(
+      of: find.text('A'),
+      matching: find.byType(ScaleTransition),
+    );
+
+    await tester.pumpAndSettle(const Duration(milliseconds: 206));
+    double scale = tester.widget<ScaleTransition>(scaleTransition).scale.value;
+    expect(scale, greaterThan(1.0));
+
+    await gesture.up();
+
+    await tester.pumpAndSettle();
+    scale = tester.widget<ScaleTransition>(scaleTransition).scale.value;
+    // Spring animations approach but don't exactly reach normal scale.
+    expect(scale, moreOrLessEquals(1.0, epsilon: 0.01));
```

Comment:
I don't think this comment is relevant after our discussion [here](https://github.com/flutter/flutter/pull/164262#issuecomment-2964631402).
```suggestion
    expect(scale, moreOrLessEquals(1.0, epsilon: 0.01));
```
Or even
```suggestion
    expect(scale, moreOrLessEquals(1.0));
```